vre/values.yaml

```diff
@@ -0,0 +1,485 @@
+      RUCIO_DISPLAY_NAME: "RUCIO - CERN VRE"
+      RUCIO_NAME: "vre-rucio.cern.ch"
+      RUCIO_SITE_NAME: "CERN"
+      RUCIO_OIDC_AUTH: "env"
+      RUCIO_OIDC_ENV_NAME: "RUCIO_ACCESS_TOKEN"
+      RUCIO_DEFAULT_AUTH_TYPE: "oidc"
+      RUCIO_OAUTH_ID: "rucio"
+      RUCIO_DEFAULT_INSTANCE: "vre-rucio.cern.ch"
+      RUCIO_DESTINATION_RSE: "CERN-EOSPILOT"
+      RUCIO_RSE_MOUNT_PATH: "/eos/eulake"
+      RUCIO_PATH_BEGINS_AT: "5" # because we are need to substitute /eos/pilot/eulake/escape/data with /eos/eulake
+      RUCIO_CA_CERT: "/certs/rucio_ca.pem"
+      OAUTH2_TOKEN: "FILE:/tmp/eos_oauth.token"
+
+    # extraFiles:
+    #   jupyter_notebook_config.json:
+    #     mountPath: /etc/jupyter/jupyter_notebook_config.json
+    #     data:
+    #       MappingKMappingKernelManager:
+    #         cull_idle_timeout: 3600
+    #         cull_interval: 600
+    #         cull_busy: false
+    #         cull_connected: true
+
+  prePuller:
+    hook:
+      enabled: true
+
+  ingress:
+    enabled: true
+    ingressClassName: null
+    annotations:
+      traefik.ingress.kubernetes.io/router.entrypoints: websecure
+      traefik.ingress.kubernetes.io/router.tls: "true"
+      #cert-manager.io/cluster-issuer: "letsencrypt" # this issues a certificate for the domain through cert-manager automatically
+    hosts: []
+      # TODO: vre chart value
+               
+reana:
+  enabled: true
+
+  postgres:
+    enabled: true
+
+  workspaces:
+    retention_rules:
+      maximum_period: forever
+      cronjob_schedule: "0 2 * * *"  # everyday at 2am
+    paths:
+      - /var/reana:/var/reana
+
+  shared_storage:
+    backend: nfs
+    volume_size: 1
+    access_modes: ReadWriteMany
+
+  secrets:
+    login:
+      iam:
+        # TODO: set this
+        consumer_key: "testkey"
+        consumer_secret: "testsecret"
+
+    database:
+      user: 
+      password:
+
+  components:
+    reana_ui:
+      enabled: true
+      local_users: false
+    reana_db:
+      enabled: false
+    reana_server:
+      environment:
+        REANA_USER_EMAIL_CONFIRMATION: false
+    # reana_workflow_controller:
+    #   image: docker.io/mdonadoni/reana-workflow-controller:0.9.1-7e03f0a-cvmfs
+    # reana_job_controller:
+    #   image: docker.io/reanahub/reana-job-controller-htcondorcern-slurmcern:0.9.1
+    reana_workflow_controller:
+      imagePullPolicy: IfNotPresent
+      image: docker.io/reanahub/reana-workflow-controller:0.9.4
+      environment:
+        SHARED_VOLUME_PATH: "/var/reana/"
+        REANA_JOB_STATUS_CONSUMER_PREFETCH_COUNT: 10
+
+
+  compute_backends:
+    - "kubernetes"
+    # - "htcondorcern"
+    # - "slurmcern"
+
+  notifications:
+    enabled: false
+    # email_config:
+    #   receiver: escape-cern-ops@cern.ch
+    #   sender: escape-cern-ops@cern.ch
+    #   login: ""
+    #   smtp_server: cernmx.cern.ch
+    #   smtp_port: 25
+
+  ingress_override: true
+
+  reana_hostname: "reana-vre.obsuks4.unige.ch"
```

Comment:
template it so that there's a hostbase path or pick a name, and jhub and reana hostnames are derived from that?